### PR TITLE
Bugfix/#19 lr scheduler

### PR DESCRIPTION
**Pull request by soeque1**
# Pull Request
레파지토리에 기여해주셔서 감사드립니다.

해당 PR을 제출하기 전에 아래 사항이 완료되었는지 확인 부탁드립니다:
- [x] 작성한 코드가 어떤 에러나 경고없이 빌드가 되었나요?
- [x] 충분한 테스트를 수행하셨나요?

## 1. 해당 PR은 어떤 내용인가요?

LR scheduler Update and Model v0.2.3

### CODE REVIEW

**Comment by soeque1** [comment]
학습 로그를 봤는데 lr-scheduler가 step 단위로 적용되어야하는데, epochs 단위로 적용되었습니다 ㅠ

File: train.py [hunk 1]
```diff
@@ -94,29 +100,62 @@ def validation_step(self, batch, batch_idx):
...
-        scheduler = {
-            "scheduler": lr_scheduler,
-            "reduce_on_plateau": True,
-            # val_checkpoint_on is val_loss passed in as checkpoint_on
-            "monitor": "val_checkpoint_on",
-        }
-        return [optimizer], [scheduler]
+        # https://github.com/huggingface/transformers/blob/a75c64d80c76c3dc71f735d9197a4a601847e0cd/examples/contrib/run_openai_gpt.py
+        param_optimizer = list(self.model.named_parameters())
+        no_decay = ["bias", "LayerNorm.bias", "LayerNorm.weight"]
```
Comment (ssaru) [inline train.py:116]: 이 부분은 외부 config에서 주는게 더 좋은 방법인 것 같다는 느낌이 드는데, 어떻게 생각하시나요?

**Review by ssaru — COMMENTED** [review]
@soeque1 
형준님 코멘트 남겼습니다.
버전 변경으로 인한 코드 변경이 많아보입니다.
카페에서 당혹스러워하시던 모습이 떠오르는군요.

1.0.x대로 버전 올리시느라 고생하셨습니다. 🙇‍♂️

File: train.py (hunk 1 above)
Comment (soeque1) [reply train.py:116]: 좋은 의견 감사드립니다.

지금 해결 하지 못하고 있는 부분인데.. 
optimizer_step은 step 단위로 움직이고, configure_optimizers 은 epoch 단위로 움직이고 있습니다 ㅠ

-> configure_optimizers에서 optimizer랑 scheduler 설정하면 잘 작동하는데 문제는 epochs 단위라서...
step 단위로 하고 싶은데.. 아직 방법을 잘 모르겠네요..

방법을 해결하고 나면 제안주신대로, optimizer와 scheduler가 config에 의해 작동할 수 있도록 한 번 리팩토링 해보겠습니다.

감사합니다.

**Review by ssaru — APPROVED** [review]
@soeque1 
Approve하였습니다.
고생하셨습니다! ㅎㅎ
